core/includes/form.inc

```diff
@@ -4000,6 +4039,45 @@ function form_process_fieldset(&$element, &$form_state) {
   return $element;
 }
 
+/**
+ * Arranges fieldsets into groups.
+ *
+ * @param $element
+ *   An associative array containing the properties and children of the
+ *   fieldset. Note that $element must be taken by reference here, so processed
+ *   child elements are taken over into $form_state.
+ * @param $form_state
+ *   The $form_state array for the form this fieldset belongs to.
+ *
+ * @return
+ *   The processed element.
+ */
+function form_process_details(&$element, &$form_state) {
+  $parents = implode('][', $element['#parents']);
+
+  // Contains form element summary functionalities.
+  $element['#attached']['library'][] = array('system', 'backdrop.form');
+
+  // The .form-wrapper class is required for #states to treat DETAILS elements
+  // like containers.
+  if (!isset($element['#attributes']['class'])) {
+    $element['#attributes']['class'] = array();
+  }
+
+  // DETAILS elements do not need 'collapsible'/'collapsed' classes, since they
```

Comment:
Here's one that is MIT license: https://www.npmjs.com/package/details-polyfill

Reply:
Thanks @jenlampton 👍🏼  ...added with [`d8e51d6` (#3629)](https://github.com/backdrop/backdrop/pull/3629/commits/d8e51d6b2b82ce2310677de01766a3536c9ea449)